test/sym/binop_f32.wat

```diff
@@ -35,9 +35,8 @@
     (f32.eq (f32.max (local.get $x) (f32.const 42)) (f32.const 0))
     (if (then unreachable))
 
-    ;; File "src/symbolic_value.ml", Assertion failed
-    ;; (f32.eq (f32.copysign (local.get $x) (f32.const 42)) (f32.const 0))
-    ;; (if (then unreachable))
+    (f32.eq (f32.copysign (local.get $x) (f32.const 42)) (f32.const 0))
+    (if (then unreachable))
```

Comment:
> This test will never trigger because the case $x = 0 has been ruled out above. It is worth asking if this is the behavior we want or not for this test file.

In this PR https://github.com/OCamlPro/owi/pull/151/, coverage-oriented tests have been reactivated.
They're not very interesting, but they do ensure that the instructions work.

> Thanks! I'm surprised that there's no need to promote any test, could you check what is going on ?

I'm not surprised there's no "test promote" (and @krtab gave the detailed reason;)